<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,0">
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Primitives</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="selectedPoint">
            <property name="text">
             <string>Point</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="selectedLine">
            <property name="text">
             <string>Line</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="selectedPolygon">
            <property name="text">
             <string>Polygon</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Transformations</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="2" column="2">
           <widget class="QSpinBox" name="spinBox_5"/>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="spinBox_3"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="rotateLabel">
            <property name="text">
             <string>Rotate</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="scaleLabel">
            <property name="text">
             <string>Scale</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QSpinBox" name="spinBox_2"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="translateLabel">
            <property name="text">
             <string>Translate</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="spinBox_4"/>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBox"/>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="applyButton">
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="PaintCanvas" name="canvas">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTableWidget" name="primitivesList"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PaintCanvas</class>
   <extends>QFrame</extends>
   <header>paintcanvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
